zookeeper-metrics-providers/zookeeper-metrics-api/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <!--
+  /**
+   * Licensed to the Apache Software Foundation (ASF) under one
+   * or more contributor license agreements.  See the NOTICE file
+   * distributed with this work for additional information
+   * regarding copyright ownership.  The ASF licenses this file
+   * to you under the Apache License, Version 2.0 (the
+   * "License"); you may not use this file except in compliance
+   * with the License.  You may obtain a copy of the License at
+   *
+   *     http://www.apache.org/licenses/LICENSE-2.0
+   *
+   * Unless required by applicable law or agreed to in writing, software
+   * distributed under the License is distributed on an "AS IS" BASIS,
+   * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   * See the License for the specific language governing permissions and
+   * limitations under the License.
+   */
+  -->
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+      <groupId>org.apache.zookeeper</groupId>
+      <artifactId>zookeeper-metrics-providers</artifactId>
+      <version>3.10.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>zookeeper-metrics-api</artifactId>
+  <packaging>jar</packaging>
+  <name>Apache ZooKeeper - Metrics API</name>
+  <description>ZooKeeper Metrics API</description>
+
+  <dependencies>
+      <dependency>
+          <groupId>ch.qos.logback</groupId>
+          <artifactId>logback-core</artifactId>
+      </dependency>
+      <dependency>
+          <groupId>ch.qos.logback</groupId>
+          <artifactId>logback-classic</artifactId>
```

Comment:
we don't need this in an API module, this is a specific implementation of a logging framework

Reply:
Addressed, the PromethuesProvider will use RateLogger, So move RateLogger from `zookeeper-server` to `metrics-api`. In RateLogger, it will use Logger, so we need slf4j-api dependency.